test/fixtures/04-with-parameter/api/Cargo.toml.backup

```diff
@@ -0,0 +1,9 @@
+[package]
```

Comment:
This file can probably be deleted, no? Or is there a reason a file with a `.backup` extension is included?

Reply:
Good catch it was leftover from my tests.